Off-Top-App/off-top-python
# OT-aggregateTopics | created a restful api to aggregate users id's and their topics

**What I did:**

- created a restful API to aggregate user ids and their topics by utilizing the API 'get-all-sessions'

- got rid of duplicate topics per user id

**How to test:**

- You can use `http://localhost:5000/get-user-topic` on postman or visit it in your browser to see the list of JSON objects that appear.
